app/services/content-service/logging.js

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Of the six different process types we have roughly 3 or 4 different logging configurations. Sharing some of the code is possible but most of it already is. I don't see this changing much except in ways that makes the processes differ more.

bunyan also doesn't seem to allow configuring a logger after creation.